Mathlib/Algebra/GroupWithZero/WithZero.lean

```diff
@@ -402,6 +408,7 @@ def expEquiv : G ≃ (Gᵐ⁰)ˣ := Multiplicative.ofAdd.trans unitsWithZeroEqui
 def logEquiv : (Gᵐ⁰)ˣ ≃ G := unitsWithZeroEquiv.toEquiv.trans Multiplicative.toAdd
 
 @[simp] lemma logEquiv_symm : (logEquiv (G := G)).symm = expEquiv := rfl
+
 @[simp] lemma expEquiv_symm : (expEquiv (G := G)).symm = logEquiv := rfl
```

Comment:
I personally like having pairs of one-liners lined up, to show they are the same